nima/testing/junit5/webserver/src/test/java/io/helidon/nima/testing/junit5/webserver/TestServerTest.java

```diff
@@ -63,6 +63,11 @@ void testSocketClientInjectedParameter(SocketHttpClient socketClient) {
         assertThat(socketClient, notNullValue());
     }
 
+    @Test
```

Comment:
There should be a test that combines an annotated client, not annotated client, and annotated with `@default` socket.
The not annotated and `@default` should return the same instance of http client.
The annotated should return a different instance pointing to the named socket

Reply:
Should I introduce an additional `@Default` annotation, or just keep non–annotated value as default?

Reply:
Fixed